ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -141,6 +143,82 @@ export default class ExperimentalTab extends PureComponent {
     );
   }
 
+  renderSecurityAlertsToggle() {
+    const { t } = this.context;
+
+    const { securityAlertsEnabled, setSecurityAlertsEnabled } = this.props;
+
+    return (
+      <>
+        <Typography
+          variant={TypographyVariant.H4}
+          color={TextColor.textAlternative}
+          marginBottom={2}
+          fontWeight={FONT_WEIGHT.BOLD}
+        >
+          {t('security')}
+        </Typography>
```

Comment:
`<Typography>` is deprecated in favor of `<Text>`. Can we replace the instances in this PR?